playbooks/bootstrap-cluster.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Test 'cephadm_bootstrap' module
+  hosts: admin
+  gather_facts: false
+  become: true
```

Comment:
what is the actual usage of `become` can we have more user friendly arg names?

Reply:
`become` is ansible parameter to execute task with `root` previllage. Refer - https://docs.ansible.com/ansible/latest/user_guide/become.html#id2